roles/servicetelemetry/tasks/component_alertmanager.yml

```diff
@@ -35,5 +35,63 @@
   when:
     servicetelemetry_vars.alerting.alertmanager.receivers.snmp_traps.enabled and
     servicetelemetry_vars.alerting.enabled
+# <--
 
-# TODO: Create route to access Alertmanager (conditional, default off)
+- name: Construct oauth redirect reference
```

Comment:
For info on the oauth annotations and use of routes: https://docs.openshift.com/container-platform/3.11/architecture/additional_concepts/authentication.html#service-accounts-as-oauth-clients